ext/scripts/tests/install_scripts/test_files/apt/with_versions/all_versions_specified

```diff
@@ -1,2 +1,2 @@
-curl|7.58.0-2ubuntu3.16 # t1
+curl|7.58.0-2ubuntu3.17 # t1
```

Comment:
Is there a specific reason why you did not go to
> [focal (20.04LTS)](https://packages.ubuntu.com/focal/curl) (web): command line tool for transferring data with URL syntax
> 7.68.0-1ubuntu2.10 [security]: amd64 i386
> 7.68.0-1ubuntu2 [ports]: arm64 armhf ppc64el s390x

https://packages.ubuntu.com/search?keywords=curl

Reply:
Is this based on Ubuntu 18.04 LTS?

Reply:
This is a test of an installation script and I guess the docker container is still based on 18.04, this doesn't build an actual script-language container